compiler/crates/relay-schema/src/relay-extensions.graphql

```diff
@@ -3,7 +3,7 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-directive @relay_test_operation on QUERY | MUTATION | SUBSCRIPTION
+directive @relay_test_operation(mock_client_data: Boolean = false) on QUERY | MUTATION | SUBSCRIPTION
```

Comment:
I experimented around with the default value here, not sure if it was working. Is the purpose more to show intent?